### PR TITLE
Feature/progress bar

### PR DESCRIPTION
**Pull request by nikokreissl**
- Added progress bar component
- Added component to `objective card on home, objective detail and to each challenge`
- Empty state shown if no challenge has been added to objective yet

### CODE REVIEW

File: components/Competition/GameDetail/index.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ import {
 
 export default function GameDetail({
   game,
-  gameNumber,
+  gamenumber,
```
Comment (actyralf) [inline components/Competition/GameDetail/index.js:19]: This should be camel case. Why did you change it?

File: components/GeneralComponents/Navigation/index.js [hunk 2]
```diff
@@ -88,18 +88,18 @@ const StyledLink = styled(Link)`
...
   padding: 10px;
-  border: ${({ linkName, active }) =>
-    linkName === active ? "1px solid var(--orange)" : "1px solid transparent"};
+  border: ${({ linkname, active }) =>
+    linkname === active ? "1px solid var(--orange)" : "1px solid transparent"};
   border-radius: 25px;
 `;
 
-function TacticIon(linkName, active) {
+function TacticIon(linkname, active) {
```
Comment (actyralf) [inline components/GeneralComponents/Navigation/index.js:99]: This should be `TacticIcon`, right?

File: components/Objectives/ObjectiveCard/ObjectiveCard.test.js [hunk 3]
```diff
@@ -30,5 +30,5 @@ test("Objective Card renders the details correctly", () => {
   render(<ObjectiveCard objective={objective} />);
 
   const objectiveName = screen.getByRole("heading", { name: "Objective Name" });
-  const numbeOfChallenges = screen.getByText("Challenges finished: 0 / 3");
+  const numbeOfChallenges = screen.getByText("Challenges completed: 0 / 3");
```
Comment (actyralf) [inline components/Objectives/ObjectiveCard/ObjectiveCard.test.js:33]: Should be `numberOfChallenges`.

**Review by actyralf — CHANGES_REQUESTED** [review]
Looks very good - only a few minor comments on naming conventions and typos.

File: components/Competition/GameDetail/index.js (hunk 1 above)
Comment (nikokreissl) [reply components/Competition/GameDetail/index.js:19]: There was one warning coming from react on a `Link` where I was passing it as camel-case and I wanted to change it just there but it seems I changed it on all places accidentally. I will revert it and just fix it on this place 🙂
Thanks for the hint!

File: components/Objectives/ObjectiveCard/ObjectiveCard.test.js (hunk 3 above)
Comment (nikokreissl) [reply components/Objectives/ObjectiveCard/ObjectiveCard.test.js:33]: Same here 🙂

**Comment by nikokreissl** [comment]
> Reference in

Thanks a lot and thank you for the feedback! 🙂

**Review by stefankorder — APPROVED** [review]
LGTM!

**Review by actyralf — APPROVED** [review]
Nice!
